src/gtkcompat.h

```diff
@@ -20,7 +20,8 @@
 
 /* Compatibility macros to support older GTK+ versions
  * 
- * TODO: This file can be removed once the code is updated to use GTK3 keysyms.
+ * Although this file might not be useful anymore, it might be used by plugins
+ * so it cannot be removed without care.
```

Comment:
Also, if we introduce GTK4 compatibility, I wouldn't be surprised this file popping up again.

Reply:
Maybe just deprecate GTK2 contents for later removal, and add GTK4 ones when they come along.